internal/pkg/skuba/deployments/ssh/gangway.go

```diff
@@ -66,3 +70,35 @@ func gangwayDeploy(t *Target, data interface{}) error {
 	_, _, err = t.ssh("kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f /tmp/gangway.d")
 	return err
 }
+
+const (
+	gangwaySecretName = "oidc-gangway-cert"
+	gangwayPodLabel   = "app=oidc-gangway"
+)
+
+func gangwayRenewCertificate(t *Target, data interface{}) error {
+	client, err := kubernetes.GetAdminClientSet()
+	if err != nil {
+		return errors.Wrap(err, "could not get admin client set")
+	}
+
+	_, err = client.CoreV1().Secrets(metav1.NamespaceSystem).Get(gangwaySecretName, metav1.GetOptions{})
```

Comment:
Is it possible to call `gangway.CreateCert()` directly without delete cert/key first?